### ISSUE TITLE
ITK-4.13.2 incompatible with GCC-9.1.0 ?

### ISSUE THREAD

**Issue by TheChymera** [InsightSoftwareConsortium/ITK#909]
### Description

ITK-4.13.2 won't build, with `Dunno about this gcc` error.

### Steps to Reproduce

```
cmake --no-warn-unused-cli -C /var/tmp/portage/sci-libs/itk-4.13.2/work/itk-4.13.2_build/gentoo_common_config.cmake -G Unix Makefiles -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DITK_USE_SYSTEM_DCMTK=ON -DITK_USE_SYSTEM_DOUBLECONVERSION=ON -DITK_USE_SYSTEM_GCCXML=ON -DITK_USE_SYSTEM_HDF5=ON -DITK_USE_SYSTEM_JPEG=ON -DITK_USE_SYSTEM_PNG=ON -DITK_USE_SYSTEM_SWIG=ON -DITK_USE_SYSTEM_TIFF=ON -DITK_USE_SYSTEM_ZLIB=ON -DITK_BUILD_DEFAULT_MODULES=ON -DITK_COMPUTER_MEMORY_SIZE=1 -DWRAP_ITK_JAVA=OFF -DWRAP_ITK_TCL=OFF -Ddouble-conversion_INCLUDE_DIRS=/usr/include/double-conversion -Ddouble-conversion_LIBRARIES=-ldouble-conversion -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF -DITK_USE_REVIEW=OFF -DITKV3_COMPATIBILITY=OFF -DVNL_CONFIG_ENABLE_SSE2=ON -DITK_WRAP_PYTHON=OFF -DCMAKE_BUILD_TYPE=Gentoo -DCMAKE_INSTALL_DO_STRIP=OFF -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/sci-libs/itk-4.13.2/work/itk-4.13.2_build/gentoo_toolchain.cmake  /var/tmp/portage/sci-libs/itk-4.13.2/work/InsightToolkit-4.13.2
```

### Expected behavior

it should compile

### Actual behavior

[build.log](https://github.com/InsightSoftwareConsortium/ITK/files/3194363/build.log)

### Reproducibility

always

### Environment

Gentoo Linux
GCC: x86_64-pc-linux-gnu-9.1.0
Python 2.7.16 and 3.6.8

Btw, does ITK-4.13.2 support python3?

**Comment by thewtex** [comment]
Addressed in #911 

> Btw, does ITK-4.13.2 support python3?

Yes.

**Comment by TheChymera** [comment]
@thewtex thank you. Can you make this fix available in some 4.* series release? This is [apparently needed as a dependency for ANTs](https://github.com/ANTsX/ANTs/issues/612#issuecomment-493694879).

**Comment by thewtex** [comment]
Yes, it will be available in 4.13.3, but it will be a while before it is released.

For GCC 9, ITK 5.0.0 is recommended -- it will be tagged this week.
